### PR TITLE
Add anchor for powered-by

### LINKED ISSUE #40: anchor supported by section
Currently the way to go to the supported by section is to go https://dask.org/ then scroll to the bottom.

A hyperlink to this section would be beneficial and perhaps hyperlinks to the other sections e.g. Powered by dask.

### PR DESCRIPTION
**Pull request by jsignell**
Closes #40

### CODE REVIEW

**Review by jrbourbeau — COMMENTED** [review]
Woo, thanks @jsignell! Would you mind adding an anchor for the "Supported By" section too? I think that's specifically what @raybellwaves  was wanting over in https://github.com/dask/dask.github.io/issues/40

P.S. TIL there's a `link-unstyled` class : )

File: index.html [hunk 1]
```diff
@@ -519,7 +519,7 @@ <h4>tsfresh</h4>
           <section id="supporters">
 
             <div class="container supporters">
-              <h2>Supported By</h2>
+              <h2 id="powered-by"><a href="#supported-by" class="link-unstyled">Supported By</a></h2>
```
Comment (raybellwaves) [inline index.html]: should that be id="supported-by"?

Comment (jsignell) [reply index.html]: yes :roll_eyes:

**Review by jrbourbeau — APPROVED** [review]
Thanks @jsignell!

**Pull request merged**